packages/react-scripts/template/src/index.js

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+// Include the default polyfills to enable modern javascript features
+// in legacy browsers. Learn more: http://bit.ly/2Bt1Yjk
+import './polyfills';
```

Comment:
this import needs to be before React, so the user don't have to move it if they want to include Map, Set, and RAF polyfill for React 16